.github/workflows/build.yaml

```diff
@@ -75,10 +77,16 @@ jobs:
         run: |
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git config user.name "github-actions[bot]"
-
       #   -- Version Bumping --
+      - name: Manual execution means no version bump
+        # If triggered by workflow dispatch, no version bump
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          echo "TARGET_ENV=${{ github.event.inputs.venue }}" >> $GITHUB_ENV
+
+          TARGET_ENV=${{ github.event.inputs.venue }}
       - name: Bump alpha version
```

Comment:
Not sure what the point of this step is

Reply:
This specifies the environment name (SIT or UAT) during a workflow dispatch.  During a regular run triggered by push, the value will be set automatically.  For workflow dispatch, I need the freedom to push any commit/branch to any environment